hive-metastore/src/test/java/org/apache/iceberg/hive/TestHiveCatalog.java

```diff
@@ -1031,14 +1052,14 @@ public void testSetDefaultPartitionSpec() throws Exception {
 
     try {
       Table table = catalog.buildTable(tableIdent, schema).create();
-      Assert.assertFalse(
-          "Must not have default partition spec",
-          hmsTableParameters().containsKey(TableProperties.DEFAULT_PARTITION_SPEC));
+      Assertions.assertThat(hmsTableParameters())
+          .doesNotContainKey(TableProperties.DEFAULT_PARTITION_SPEC)
+          .as("Must not have default partition spec");
```

Comment:
the .as(..) part always need to come before the last assertion

Reply:
Changed.